modules/mod_ginger_base/lib/js/base.js

```diff
@@ -1,99 +1,60 @@
-(function($) {
```

Comment:
Will removing this not cause any timing issues when JQuery hasn’t yet been fully loaded?

Reply:
Will check on this